### ISSUE TITLE
Is there a binary out there?

### ISSUE THREAD

**Issue by rufus42** [willcrichton/mdbook-quiz#12]
Hello @willcrichton,

sounds great!
Unfortunately I can't install Rust.
Is there a binary out there?

Thanks & greetings
rufus42

**Comment by willcrichton** [comment]
No, right now mdbook-quiz is only distributed through crates.io.

But how are you using mdBook in the first place? Don't you also need to `cargo install mdbook`?

**Comment by willcrichton** [comment]
I just added support for releasing pre-built binaries. Currently it doesn't support Windows because apparently `npm install --global` has different behavior in Windows CI runners than other operating systems. I will leave this issue open until I figure out how to support Windows.

**Comment by rufus42** [comment]
Hello @willcrichton,

thank you very much for the binaries.
I hope they help Linux and Apple users.

Personally, of course, I would be happy to see a Windows binary, but regardless: thank you very much for your response.

Many greetings
rufus42
